jetbrains-rider/build.gradle

```diff
@@ -63,23 +72,25 @@ intellij {
 tasks.withType(PrepareSandboxTask).configureEach {
     dependsOn buildReSharperPlugin
 
-    from(buildReSharperPlugin.outputs, {
+    def files = resharperParts.collect {"$resharperBuildPath/bin/$it/$buildConfiguration/${it}.dll"} +
+        resharperParts.collect {"$resharperBuildPath/bin/$it/$buildConfiguration/${it}.pdb"}
+    from(files, {
         into("${intellij.pluginName}/dotnet")
     })
 }
 
-compileKotlin.dependsOn(generateModel)
+compileKotlin.dependsOn(generateModels)
 
 test {
     systemProperty("log.dir", "${project.intellij.sandboxDirectory}-test/logs")
     useTestNG()
-    environment("LOCAL_ENV_RUN", localEnv)
+    environment("LOCAL_ENV_RUN", true)
```

Comment:
What was localEnv doing before?

Reply:
its used when built on the JB intranet but its safe we always are LOCAL_ENV_RUN instead..
It has to do with the Rider test framework self bootstraping